setup.py

```diff
@@ -11,21 +11,15 @@
     description="Rust inspired Result and Option types",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="",
+    url="https://github.com/danielSanchezQ/rusty_results",
     package_data={".": ["py.typed"]},
-    packages=setuptools.find_packages(
-        ".",
-        exclude=[
-            "tests*",
-            "examples*"
-        ]
-    ),
+    packages=setuptools.find_packages(exclude=("*tests*",)),
```

Comment:
uhm, what was the magic behind this? it does search for the `rusty_results` package by name already?

Reply:
Changes' motifs:
1. The first parameter `"."` is already the default (that's why I removed it, I can undo that if you prefer the explicitness). The only package found is `rusty_results`.
2. According to the definition, `examples` is not a `python package`, hence its removal.
3. The `tests` directories contain sub-packages, that's why the `*` (star) at the right of `*tests` is required. Otherwise, it doesn't exclude those, it would only exclude the modules in the package.